.eslintrc.json

```diff
@@ -56,7 +56,8 @@
             "fixStyle": "inline-type-imports"
           }
         ],
-        "@typescript-eslint/no-unused-vars": "off"
+        "@typescript-eslint/no-unused-vars": "off",
+        "@typescript-eslint/explicit-function-return-type": "off"
```

Comment:
@razor-x I've removed the `@typescript-eslint/strict-boolean-expressions` override as it wasn't being triggered as part of _this_ PR, `@typescript-eslint/explicit-function-return-type` should remain turned off though

Reply:
I am suggesting setting it to warn for now, since I think we can fix the warnings later.